style.css

```diff
@@ -0,0 +1,53 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,500;1,600;1,700&display=swap');
+
+.main_container {
+  background-color: #e5e5e5;
```

Comment:
- [ ] Good work implementing the background image, however the background color of this section should be ```#fff``` as in the Figma design

Reply:
I have now changed it to #fff and changed the strong tag to a tag as requested.
Thank you.